packages/yew-router-macro/src/routable_derive.rs

```diff
@@ -195,6 +195,10 @@ pub fn routable_derive_impl(input: Routable) -> TokenStream {
     };
 
     quote! {
+        ::std::thread_local! {
+            static __ROUTER_CURRENT_ROUTE_CACHE: ::std::cell::RefCell<::std::option::Option<#ident>> = ::std::cell::RefCell::new(::std::option::Option::None);
```

Comment:
Are you sure this works with multiple Routables in the same module?

Reply:
I don't think it will. I'll make it use an ident specific to the enum